Neos.ContentRepository.LegacyNodeMigration/Classes/NodeDataToEventsProcessor.php

```diff
@@ -87,7 +86,7 @@ public function __construct(
         private readonly Filesystem $files,
         private readonly iterable $nodeDataRows,
     ) {
-        $this->sitesNodeTypeName = NodeTypeName::fromString('Neos.Neos:Sites');
+        $this->sitesNodeTypeName = NodeTypeName::fromString(NodeTypeNameFactory::NAME_SITES);
```

Comment:
```suggestion
        $this->sitesNodeTypeName = NodeTypeName::forSite();
```

Reply:
yes for this case i very much agree to use this ^^ i was a bit blinded.